frontend/src/site/Login.tsx

```diff
@@ -31,6 +31,31 @@ export default function Login(
     const handlePassword = (e: ChangeEvent<HTMLInputElement>) => {
         setPassword(e.target.value)
     }
+
+    const pwHasLowerLetter = () => {
+        return !!password.match("(?=.*[a-z])");
```

Comment:
Vorausgesetzt ich verstehe richtig (das du das Ergebnis in einen boolean Konvertierst), sorgt `Boolean(password.match("(?=.*[a-z])"))` ebenfalls für's richtige Ergebnis und vermeidet Stirnrunzeln :)

https://stackoverflow.com/a/784946/11515036